dnn/variational_inference.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<p>this section is unnecessary and probably superfluous for someone not familiar with python.</p><p>you are also importing in the previous code block, so its unclear to someone unfamilar that these are local imports that can be changed.</p><p> </p><p>we should just merge this into the previous section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>